RELEASES.md

```diff
@@ -8,6 +8,7 @@
 - renamed crate from `stretch2` to `sprawl`
 - updated to the latest version of all dependencies to reduce upstream pain caused by duplicate dependencies
 - renamed `stretch::node::Strech` -> `sprawl::node::Sprawl`
+- `Sprawl::remove` now returns a `Result<usize, Error>`, to indicate if the operation was sucessful, and if it was, which ID was invalidated.
```

Comment:
This should be under a 0.2.0 heading, as I don't want to publish this in 0.1 to ease the migration from stretch.

Reply:
Ah, right. Thanks!

Reply:
Fixed as of https://github.com/DioxusLabs/sprawl/pull/106/commits/8b227be403d33ea05dfe771b143821afd9e1a98e